helm/hatchery/scripts/workspaces_launch_test.py

```diff
@@ -0,0 +1,243 @@
+"""
```

Comment:
Should this script be built into a container somewhere? I don't think it's going to be pulled into a configmap or something so this file is not being used right now? Maybe I'm missing something 😬

Reply:
Yeah this script isn't being used right now. Was hoping, some time in the future, when we move away from cloud automation, i could use the script from here. Original idea was for my job to run this script, but its only available from cloud-auto at the moment. If there's another way to run this script, im open to changing how im running it right now.